doc/FAQ.md

```diff
@@ -96,5 +96,8 @@ This is a new feature of monero core to share certain key images with other mone
 Creating a file named `.nocrazypass` in the wallets folder will disable generation of crazypass for NEW passwords (new wallet or change password).
 The content of the file is not read and is irrelevant.
 Wallets with CrAzYpass will continue working normally. The currently set real wallet password can be checked in the "Show Secrets".
+The wallets folder is:
+- 1.x: `monerujo` on your external storage
```

Comment:
good idea to add this to the FAQ!

external storage is no longer used (android restrictions) - pls either remove it or comment (maybe better option) that this is a legacy folder

Reply:
That's why I added the version number ;) but fair enough, an additional legacy comment won't hurt.